CanvasColorSpaceProposal.md

```diff
@@ -186,7 +184,7 @@ Authors of games and imaging apps are expected to be enthusiastic adopters.
 
 * Should black level compensation be implied in "linear spaces", such that (0,0,0) always represents absolute black in linear color values?
 
-* Should it be possible to specify the color space parameter as an array representing a fallback list? Example: <code>canvas.getContext('2d', {colorSpace: ["p3", "rec2020"]});</code> would mean use p3 if possible; if not supported, try rec2020; and if rec2020 is not supported, fallback to srgb, which is the default.  Since color spaces are already feature detectable, this would be a convenience feature.  Same question applies to pixelFormats.
+* RESOLVED OBSOLETE(?): Should it be possible to specify the color space parameter as an array representing a fallback list? Example: <code>canvas.getContext('2d', {colorSpace: ["p3", "rec2020"]});</code> would mean use p3 if possible; if not supported, try rec2020; and if rec2020 is not supported, fallback to srgb, which is the default.  Since color spaces are already feature detectable, this would be a convenience feature.  Same question applies to pixelFormats.
```

Comment:
I agree we should remove this item. I think the only conditionally supported thing will be float16 format.  When float16 is not available the only thing to fall back to is sRGB 8-8-8-8

Reply:
Removed.